etcdserver/raft.go

```diff
@@ -85,6 +85,15 @@ type apply struct {
 	raftDone <-chan struct{} // rx {} after raft has persisted messages
 }
 
+// activeRaft represents an activated raft instance via raft.{Res,S}tartNode
+type activeRaft struct {
```

Comment:
`activeNode`? The functions returning the new structure already have a `Node` in the suffix